### PR TITLE
DB-8146 save accumulated cost of a join path in optimizable so that i…

### PR DESCRIPTION
**Pull request by yxia92**
…t can be accurately restored when retreated from a join path

### CODE REVIEW

**Comment by yxia92** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by yxia92** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/OptimizerFlag.java [hunk 1]
```diff
@@ -105,7 +105,9 @@
     COST_INCLUDING_STATS_FOR_INDEX(TRACE),
     INFEASIBLE_JOIN(DEBUG),
     SPARSE_INDEX_NOT_ELIGIBLE(TRACE),
-    HJ_NO_EQUIJOIN_COLUMNS(DEBUG);
+    HJ_NO_EQUIJOIN_COLUMNS(DEBUG),
+    REWIND_JOINORDER(TRACE),
+    PULL_OPTIMIZAL(TRACE);
```
Comment (msirek) [inline db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/OptimizerFlag.java]: Should this be PULL_OPTIMIZABLE ?

Comment (yxia92) [reply db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/OptimizerFlag.java]: Thanks @msirek for catching the typo. I've fixed it.

**Pull request merged**